student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, name = 'Unknown', age = nil, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
   end
 
   def play_hooky
```

Comment:
- [ ] It would be nice if you could create a classroom method that allows a student to belong to a class. It's required that a method that serves as the 'belongs-to' side (a student belongs to a classroom) should be created. 👍🏽